requirements.in

```diff
@@ -0,0 +1,7 @@
+# This file is part of REANA.
+# Copyright (C) 2017, 2018 CERN.
```

Comment:
2020

(we did not have this file in 2017, 2018)

Reply:
Done 👍

Reply:
You edited the wrong file, the 2020 should be in the new `requirements.in` file, not in the old `MANIFEST.in` ...